src/Okta.Sdk.IntegrationTests/FeaturesScenarios.cs

```diff
@@ -0,0 +1,61 @@
+﻿// <copyright file="FeaturesScenarios.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class FeaturesScenarios
+    {
+        [Fact]
+        public async Task ListFeatures()
+        {
+            var client = TestClient.Create();
+            var features = await client.Features.ListFeatures().ToListAsync();
+            features.Should().NotBeNull();
+            features.Count.Should().BeGreaterThan(0);
+        }
+
+        [Fact]
+        public async Task GetFeatureById()
+        {
+            var client = TestClient.Create();
+            var feature = (await client.Features.ListFeatures().ToListAsync()).FirstOrDefault();
+            feature.Should().NotBeNull();
+
+            var retrievedFeature = await client.Features.GetFeatureAsync(feature.Id);
```

Comment:
Is it possible to create a feature and test with it?

Reply:
I don't believe it would be a trivial thing to create a feature for the purposes of testing, there is no api exposed for this purpose.  Features themselves are the units of functionality that we offer to customers and the list of features is a well know/defined list.  It would potentially cause broader issues if we added features purely for testing.

Reply:
Ah! I see. So, if there's no API provided let's confirm that its exclusion was intended (I think in this case makes sense).